lib/vsc/install/shared_setup.py

```diff
@@ -1382,7 +1382,10 @@ def parse_target(self, target, urltemplate=None):
         else:
             log.info('adding prospector to tests_require')
             tests_requires = new_target.setdefault('tests_require', [])
-            tests_requires.extend(['prospector >= 0.12.1', 'pylint<2.0.0', 'pyflakes<1.7.0,>=1.5.0'])
+            tests_requires.append('prospector >= 1.1.4')
+            # Python 2.x requires pylint < 2.0, since pylint 2.0 or newer is Python 3 only
+            if sys.version_info < (2,):
+                tests_requires.append('pylint<2.0.0')
```

Comment:
hah, this is actually bullshit, since:

```
>>> sys.version_info
sys.version_info(major=2, minor=7, micro=10, releaselevel='final', serial=0)
>>> sys.version_info < (2,)
False
```

So this can just be dropped...

Reply:
fixed in #91